libshaderc_util/src/compiler.cc

```diff
@@ -171,7 +173,6 @@ bool Compiler::Compile(
   }
 
   // Parsing requires its own Glslang symbol tables.
```

Comment:
This comment is now out of place.  
It was talking about the use of the GlslInitializer object.